src/components/builder.cpp

```diff
@@ -614,5 +618,23 @@ void builder::tag_close(attribute attr) {
       break;
   }
 }
+string builder::add_surrounding_tag(const space_size& space) {
```

Comment:
`get_format_string` may be a more descriptive name since this function doesn't actually add a tag if we use spaces.